cl/recap/tasks.py

```diff
@@ -1894,6 +1894,11 @@ def fetch_pacer_doc_by_rd(
         self.request.chain = None
         return
 
+    if rd.pacer_doc_id.count("-") > 1:
```

Comment:
We do this little wonky thing somewhere else too, don't we? Can we put it in a one-liner function so that it has a name like `is_appellate_pacer_id`, or something?

Reply:
> Can we put it in a one-liner function so that it has a name like is_appellate_pacer_id, or something?

Sure, I'll add the helper function.